pkg/resourcecollector/resourcecollector.go

```diff
@@ -179,6 +179,29 @@ func resourceToBeCollected(resource metav1.APIResource, grp schema.GroupVersion,
 	return GetSupportedK8SResources(resource.Kind, optionalResourceTypes)
 }
 
+func resourceToBeCollectedWithExcludeTypes(resource metav1.APIResource, grp schema.GroupVersion, crdKinds []metav1.GroupVersionKind, optionalResourceTypes []string, exlcudeTypes []string) bool {
```

Comment:
We could reuse the function resourceToBeCollected. The only thing we have to check extra is 
```
	for _, excludeType := range exlcudeTypes {
		if strings.EqualFold(resource.Kind, excludeType) {
			return false
		}
	}
```
but for rest we could just call that function. Isn't it?